jxl/src/color/tf.rs

```diff
@@ -46,6 +46,10 @@ const PQ_INV_EOTF_P_SMALL: [f32; 5] = [
 const PQ_INV_EOTF_Q_SMALL: [f32; 5] =
     [3.371868e1, 1.477719e3, 1.608477e4, -4.389884e4, -2.072546e5];
 
+const HLG_A: f64 = 0.17883277;
```

Comment:
wdyt of putting the constants closer to the functions?
(and similarly for the PQ ones, I guess)

Reply:
Sure, moved the constants.